src/pages/landing/Landing.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect, useState } from "react";
+import { useInterval } from "usehooks-ts";
+
+import { Container, Box, Typography, Button, TextField } from "@mui/material";
+import ConnectWallet from "components/ConnectWallet/ConnectWallet";
+
+import { useAppDispatch, useAppSelector } from "store/store";
+import { fetchPrice } from "./slices/price.slice";
+import { fetchTiers } from "./slices/tiers.slice";
+
+import type { Tier } from "./slices/tiers.slice";
+
+import "./Landing.css";
+
+const REFRESH_TIME = 30;
+
+const ConnectSection = (props: { onSubmit: () => void }) => (
+  <Box className="flex flex-row-reverse flex-1">
+    <Box className="flex flex-col justify-center m-0 min-w-[50vw] bg-white">
+      <Box className="text-center">
+        <Typography variant="h3" className="font-bold mb-4 tracking-[.2em]">
+          Testing Tool
+        </Typography>
+        <Typography variant="h6" className="font-normal mb-40 text-[#A6A7AD]">
+          Connect your in-browser wallet to login/sign-up
+        </Typography>
+        <Button variant="contained" size="large" className="py-3 px-4 normal-case font-medium" onClick={props.onSubmit}>
+          Connect your wallet
+        </Button>
+        <ConnectWallet />
+      </Box>
+    </Box>
+  </Box>
+);
+
+const SubscriptionSection = (props: { tiers: Tier[], tierId: string|null, onSelect: (tierId: string) => void }) => (
+  <Box className="flex flex-col m-0 pt-24 min-w-[50vw] bg-white">
+    <Box className="text-center">
+      <Typography variant="h4" className="font-bold mb-2 tracking-[.2em]">
+        Testing Tool
+      </Typography>
+      <Typography variant="subtitle1" className="font-normal mb-16 text-[#A6A7AD]">
+        Please choose the Package you wish to subscribe to.
```

Comment:
```suggestion
        Please choose a subscription
```

I don't like the final period and I didn't like the wording. If you think this is worse, disregard it, I don't have a strong feeling about this ^^